project-base/.env

```diff
@@ -25,8 +25,12 @@ REDIS_HOST=redis
 REDIS_PREFIX=''
 
 MAILER_DSN=smtp://smtp-server:25
-MAILER_DELIVERY_WHITELIST=/@shopsys\.com$/
-MAILER_MASTER_EMAIL_ADDRESS=no-reply@shopsys.com
+# should be set to 0 in production
```

Comment:
I think this comment is not fair for other variables that have none :)

Reply:
I would set this value to 0 in env.prod instead ;)

Reply:
it should be for a production running shop, not a production environment... for example, devel is also a production environment, but not a production running shop. 

and I believe for comments for all env variables :D

Reply:
it should be for a production running shop, not a production environment... for example, devel is also a production environment, but not a production running shop. 

and I believe for comments for all env variables :D

Reply:
reworked in 25fae236c